tests/test_smoketest.py

```diff
@@ -124,94 +67,34 @@ def smoketest(self, starting_replica, checkout_bucket, test_bucket):
             with self.subTest(f"{starting_replica.name}: Create a subscription for replica {replica} using the "
                               f"query: {query}"):
                 notification_key = f'notifications/{uuid.uuid4()}'
-                url = s3.generate_presigned_url(ClientMethod='put_object',
-                                                Params=dict(Bucket=self.notification_bucket,
-                                                            Key=notification_key,
-                                                            ContentType='application/json'))
-                put_response = run_for_json([f'{self.venv_bin}hca', 'dss', 'put-subscription',
-                                             '--callback-url', url,
-                                             '--method', 'PUT',
-                                             '--es-query', json.dumps(query),
-                                             '--replica', replica.name])
+                url = BaseSmokeTest.generate_presigned_url(self, self.notification_bucket, notification_key)
```

Comment:
Is there a reason to call the base inherited class instead of self?

Meaning:
`url = BaseSmokeTest.generate_presigned_url(self, self.notification_bucket, notification_key)`
instead of:
`url = self.generate_presigned_url(self, self.notification_bucket, notification_key)`

Reply:
resolved.